docs/releases/1.32-NOTES.md

```diff
@@ -0,0 +1,45 @@
+## Release notes for kOps 1.32 series
+
+**&#9888; kOps 1.32 has not been released yet! &#9888;**
+
+This is a document to gather the release notes prior to the release.
+
+# Significant changes
+
+* TODO
+
+## Some Feature
+
+* TODO
+
+## AWS
+
+* TODO
+
+## GCP
+
+* TODO
+
+## Openstack
+
+* TODO
+
+# Other changes of note
+
+* TODO
+
+# Breaking changes
+
+## Other breaking changes
+
+* TODO
+
+# Known Issues
+
+* TODO
+
+# Deprecations
+
+* Support for Kubernetes version 1.26 is deprecated and is removed in kOps 1.32.
```

Comment:
I _think_ this is right, we should actually remove 1.26 support in 1.32 (1.N in 1.N+6), and we should announce the deprecation of the next k8s version in the next kOps version.  (?)

Reply:
Yes, I calculated earlier 1.27 to be removed in 1.33.